dependencies.env

```diff
@@ -0,0 +1 @@
+FFI_VERSION=0.5.1
```

Comment:
0.5.2 is latest now, test should pass after #250 merges and you rebase on development